libphobos/m4/druntime/libraries.m4

```diff
@@ -57,10 +57,91 @@ AC_DEFUN([DRUNTIME_LIBRARIES_ZLIB],
 
 # DRUNTIME_LIBRARIES_ATOMIC
 # -------------------------
-# Allow specifying whether to use libatomic for
-# 128 bit atomic support.
 AC_DEFUN([DRUNTIME_LIBRARIES_ATOMIC],
 [
+  # Check for atomic builtins up to 64 bit.
```

Comment:
OK.  By the way, added boards + scripts to run testsuite under qemu (only arm for now).

https://github.com/D-Programming-GDC/buildbot-gdc/commit/c75d301e80c03249dde077510a28267e48efb2c4

Reverting the "disable runnable tests" patch.  Also fixed build for armhf - the absolute minimal needed should be setting `--with-fpu`, but setting both arch, fpu and float flags explicitly on both hard and soft-float targets just to be sure (https://github.com/D-Programming-GDC/buildbot-gdc/commit/b99413a852ef1181868abeb1b68ff11fb3d690cd)